src/transport/raw/TCP.cpp

```diff
@@ -125,6 +125,9 @@ void TCPBase::Close()
         mListenSocket->Free();
         mListenSocket = nullptr;
     }
+
+    CloseActiveConnections();
```

Comment:
Seems like the destructor should call Close() instead of us copy/pasting everything between them?